.gitignore

```diff
@@ -4,3 +4,6 @@ __pycache__
 workflows/.snakemake/*
 workflows/logs_slurm/*
```

Comment:
@andrewkern where should we run `snakemake`? `analysis2` or `analysis2/workflows`?

Reply:
`analysis2` is the place. we might move the `config` directory up to that level too